.github/workflows/check_cpln_links.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Links
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  check_cpln_links:
+    runs-on: ubuntu-latest
+    name: Check Links
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Validate outgoing links to Control Plane
+        run: |
+          TERM=xterm-color ./script/check_cpln_links
+        shell: bash
```

Comment:
Is this required?

It does say [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) that if `bash` is not specified explicitly and it's not found in the path, it treats it as `sh` instead, but does this mean that we don't have `bash` in the path here?

I guess just remove if not needed.

Reply:
> Is this required?

Dunno. The example for running a Bash command includes it, and I personally use `fish` so after running into enough bugs due to shebang-less scripts, I've gotten into the habit of being explicit about which shell I'm expecting any time I'm running something.

I prefer keeping the explicit shell selection, but if you feel strongly about it, let me know and I can drop it.